File: quick-build/corpus/apps/hello-java/src/main/java/org/appdevforall/cotg/corpus/hellojava/DetailActivity.java
package org.appdevforall.cotg.corpus.hellojava;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

public class DetailActivity extends Activity {

	public static final int ID_DETAIL = 2002;

	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);

		TextView view = new TextView(this);
		view.setId(ID_DETAIL);
		view.setText("Detail screen: " + new Greeter().greet("Detail visitor"));
		setContentView(view);
	}
}

File: quick-build/corpus/apps/hello-kotlin/src/main/java/org/appdevforall/cotg/corpus/hellokotlin/DetailActivity.kt
package org.appdevforall.cotg.corpus.hellokotlin

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class DetailActivity : Activity() {
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)

		val view = TextView(this)
		view.id = ID_DETAIL
		view.text = "Detail screen: " + Greeter().greet("Detail visitor")
		setContentView(view)
	}

	companion object {
		const val ID_DETAIL = 1002
	}
}
